draft-ietf-masque-connect-udp-listen.md

```diff
@@ -107,11 +107,38 @@ each HTTP Datagram (see {{fmt-dgram-uncomp}}), or registered via capsules and
 then compressed (see {{fmt-capsule-assign}}).
 
 When performing URI Template Expansion of the UDP Proxying template (see
-{{Section 3 of CONNECT-UDP}}), the client sets both the "target_host" and the
-"target_port" variables to the '\*' character (ASCII character 0x2A). Note that
-the '\*' character MUST be percent-encoded before sending, per {{Section 3.2.2
+{{Section 3 of CONNECT-UDP}}), the client follows the same template as
+CONNECT-UDP and sets the "target_host" and the "target_port" variables
+to one of its targets. It adds the connect-udp-bind header as specified in
+{{hdr}} to indicate that it prefers bind but would accept regular
+CONNECT-UDP to the target in case the proxy does not support it.
+If the proxy supports CONNECT-UDP Bind, it returns the connect-udp-bind
+response header value set to true and the proxy allocates context ID 0
+to the target and compressed datagrams {#compressed-operation} are transmitted
+between the client and proxy. On the other hand, if the proxy does not support
```

Comment:
I don't think we need this last sentence here - we're just explaining the concept, no need to say what the proxy should do if it doesn't support it